### PR TITLE
Capture metadata on signup

### PR DESCRIPTION
**Pull request by kpthatsme**
## Changes
Sends the `analytics_metadata` object for users and orgs as properties for sign up events.

Note: cloud tests are failing since they need to be updated in that repo. 

The failures are due to additional properties coming back from this object. I've switched the test to check for a few key properties (the ones being checked historically) instead of the entire dictionary, checking each property makes updating instrumentation a lot more involved than it should be most cases.

## How did you test this code?
- Hit the signup flows locally
- Updated the unit tests (cloud still needs to be updated)

### CODE REVIEW

File: posthog/models/organization.py [hunk 1]
```diff
@@ -155,6 +155,7 @@ def get_analytics_metadata(self):
             "person_count": sum((team.person_set.count() for team in self.teams.all())),
             "setup_section_2_completed": self.setup_section_2_completed,
             "personalization": self.personalization,
+            "name": self.name,
```
Comment (samwinslow) [inline posthog/models/organization.py:158]: Nice! Adding the name here will be super useful later.

**Review by samwinslow — APPROVED** [review]

**Comment by kpthatsme** [comment]
@EDsCODE I'm going to need you to merge this in for me please.

**Comment by paolodamico** [comment]
Curious on the rationale for this. I'm particularly interested to see if we're not patching something that should be more broadly fixed for our users (like properties at a point in time).

**Comment by kpthatsme** [comment]
> Curious on the rationale for this. I'm particularly interested to see if we're not patching something that should be more broadly fixed for our users (like properties at a point in time).

We need this metadata in this event as we're going to use this event for our lead pipeline. Re: privacy – we should be respecting both the opt-out and the anonymization still.

**Comment by EDsCODE** [comment]
I think you need a posthog-cloud complementary PR to update payload checks such as [this](https://github.com/PostHog/posthog-cloud/blob/master/multi_tenancy/tests/test_signup.py#L62) so that the posthog-cloud tests will pass

**Pull request merged**